### DISCUSSION TITLE
I like 2FA, but I don't like enabling bluetooth on my browser.

### DISCUSSION THREAD

**Discussion by erikh2000** [community/community#137107] [npm]
I dislike granting bluetooth access to any webpage. Does the NPM login 2FA really need to use this?

My request is to allow 2FA to use some of the boring old-fashioned messaging like email and SMS. Maybe I've missed an option somewhere.

**Comment by radar-blip** [comment]
From your profile page, go to `Settings`, then to `Password and Authentication` scroll down and you will see your 2FA options there.

![thiss](https://github.com/user-attachments/assets/c724677a-7946-4187-9e86-003b284b15ab)
